### ISSUE TITLE
why always 'Found 0 boxes for img' in my model tested?

### ISSUE THREAD

**Issue by sunyhaime** [qqwweee/keras-yolo3#553]

**Comment by ChristianBlum** [comment]
I had the same problem. The reason was that my detection accuracy was very low. Try to set the score in yolo.py to 0.01. 

Dose someone know why my detection score is that low. I trained about 150 epochs and then there was an early stopping. The loss was still at 13. And if I test my model I will find some correct detections with the probability of e.g. 1.7%
@qqwweee

**Comment by ujwalv** [comment]
@ChristianBlum that's a great tip, now it detects all the object, care to explain what exactly the SCORE value does ?

**Comment by ChristianBlum** [comment]
As far as I understood it right, with the score value you can filter out some detections with a specific detection probability. 
If you say score = 0.3, all the detections which have a detection probability lower than 0.3 are not shown in the picture.
